src/main/java/hudson/plugins/git/GitChangeSet.java

```diff
@@ -352,12 +352,28 @@ String getParentCommit() {
     /**
      * Returns user of the change set.
      *
+     * @deprecated  Use {@link #findOrCreateUser(String,String,boolean,boolean)}
```

Comment:
Could you follow the Javadoc guidlines (https://www.oracle.com/technetwork/java/javase/documentation/index-137868.html#orderoftags) and put `@deprecated` at the end?